src/popup/components/ad-hoc-tools-panel.tsx

```diff
@@ -24,7 +24,8 @@ const toggleShouldNotHaveDivider = (
 };
 
 export const AdHocToolsPanel = NamedFC<AdHocToolsPanelProps>('AdHocToolsPanel', props => {
-    let addNewRow: boolean;
+    let newStyle: string = styles.noRowNeeded;
```

Comment:
You can likely move this inside the `getTogglesWithDividers` method; removes what ends up being a somewhat unnecessary "stateful"-like property.

Reply:
Never mind, I totally misread what's happening here.